core/common/src/main/kotlin/app/k9mail/core/common/oauth/InMemoryOauthConfigurationProvider.kt

```diff
@@ -0,0 +1,18 @@
+package app.k9mail.core.common.oauth
+
+internal class InMemoryOauthConfigurationProvider(
+    private val oauthConfigurationFactory: OAuthConfigurationFactory,
```

Comment:
Both `oauthConfigurationFactory` and `oAuthConfigurationFactory` are kinda ugly. We can probably drop the `oauth` prefix.

```suggestion
    private val configurationFactory: OAuthConfigurationFactory,
```